### DISCUSSION TITLE
실행 시에 에러가 납니다

### DISCUSSION THREAD

**Discussion by konzess** [PeterSuh-Q3/tinycore-redpill#36] [Q&A]
![tcrp error](https://github.com/PeterSuh-Q3/tinycore-redpill/assets/73073115/695bc652-6a0e-4cc9-a95e-e286850138b6)

Proxmox VE 에 vm 만드는 방식으로 TCRP 빌드 및 설치하고 난 이후,
재부팅 여러번 시도했는데,
그 중 30 ~ 40% 정도는 잘 부팅이 되고 있습니다.
하지만 그 중의 60 ~ 70% 는 부팅 시마다 위와 같은 에러 메시지가 나타나면서 부팅이 안되고 있거든요

디스크 크기, 램, cpu 할당 개수 등을 여러가지 조건으로 바꿔봐도 동일합니다...

다른 설정 다 그대로 둔 채로, 그냥 재부팅만 반복해서 했을 뿐인데,
동일한 설정 상태에서 언제는 부팅 잘 되고, 언제는 안되고...
왜 그러는 걸까요.....ㅜ

해결하려면 어떻게 해야 될까요..?

**Comment by PeterSuh-Q3** [comment]
지금은 해결된 이슈라서 이 토의는 닫겠습니다.
감사합니다.
